eclipse-bluechi/README.md

```diff
@@ -0,0 +1,206 @@
+
+* `{SERVICE_NAME}.kube`: Used by systemd to point to a Kubernetes resource definition
+  containing the workload definition.
+
+  Example of `freyja.kube`:
+    ```kube
+    # https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html
+    [Kube]
+    Yaml=freyja.yml
+
+    # Commented to disable the service to automatically start
+    # [Install]
+    # WantedBy=default.target
+    ```
+* `{SERVICE_NAME}.yaml`: A Kubernetes resource definition (either `v1.Pod` or
+  `apps/v1.Deployment`) that describes the workload.
+
+  Example of `freyja.yml`:
+    ```yaml
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: freyja
+      name: freyja
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: freyja
+      template:
+        metadata:
+          labels:
+            app: freyja
+        spec:
+          hostNetwork: true
+          containers:
+            - name: local
+              image: sdvblueprint.azurecr.io/sdvblueprint/eclipse-freyja/local-with-ibeji:0.1.0
+              imagePullPolicy: IfNotPresent
+    ```
+
+If you edit the source code of a component then build and push an image of it to your container registry, you will need to edit the corresponding `{SERVICE_NAME}.yaml` file in the `/etc/containers/systemd` directory. The value of the `image` field in the `{SERVICE_NAME}.yaml` file should point to the image in your container registry.
+
+Creating, changing or updating a file in `/etc/containers/systemd` requires you to run `systemctl daemon-reload` afterwards to generate the corresponding systemd unit files in
+`/run/systemd/generator`.
+
+### Service Lifecycle
+
+Services can be managed by using `systemctl`, systemd's administrative CLI.
+
+Starting, stopping, restarting services is as easy as:
+
+* `systemctl stop {SERVICE_NAME}`
+* `systemctl start {SERVICE_NAME}`
+* `systemctl restart {SERVICE_NAME}`
+
+> Make sure to run `systemctl daemon-reload` in case something changed in either Quadlet or systemd unit files.
+
+### Monitoring and Logs
+
+BlueChi's CLI (`bluechictl`), can be used to retrieve information from
+managed nodes:
+<https://github.com/eclipse-bluechi/bluechi/blob/main/doc/man/bluechictl.1.md>.
+
+#### Using Systemctl
+
+Simply run `systemctl status {SERVICE_NAME}` where `{SERVICE_NAME}` is the name of your .kube file.
+
+#### Using journalctl
+
+This is valid for any systemd defined service, simply run `journalctl -xeu {SERVICE_NAME}`
+
+#### Podman
+
+You can also list all active containers by running `podman ps` and then `podman logs {CONTAINER_NAME_OR_ID}` to
+get logs from the container using podman. Replace `{CONTAINER_NAME_OR_ID}` with the container's name or ID.
+
+## Running the Smart Trailer Example with BlueChi's devcontainer
+Inside of the [devcontainer](#two-development-environments):
+1. Follow the instructions in [Starting All the Services](#starting-all-the-services) to start up the in-vehicle stack.
```

Comment:
It doesn't look like these two links lead to anywhere
"#two-development-environments" and "#starting-all-the-services"

Reply:
ah, just left overs from the previous doc. Let me remove them.